src/status_im/data_store/realm/schemas/base/account.cljs

```diff
@@ -239,3 +239,6 @@
                  {:syncing-on-mobile-network? {:type :bool :default false}
                   :remember-syncing-choice?   {:type :bool :default false}}))
 
+(def v22 (update v21 :properties merge
```

Comment:
You could stick it into `settings` and it would not require a migration, but either places are fine with me

Reply:
oh ok, I didn't know we have such field

Reply:
fixed